revision-1/lima/authors/19.conversation.md

```diff
@@ -0,0 +1,4 @@
+<|user|>
+Skrifaðu útdrátt á máltæknigrein sem sýnir hvernig er hægt að þjálfa spjallmenni með því að fínþjálfun forþjálfað líkan á 1000 vandlega völdum dæmum.
+<|system|>
+Mállíkön hafa sýnt góðan árangur með blöndu af viðgjafarlausu námi, [large scale instruction tuning] og styrkingarnámi. Við skoðum hlutfallslegt mikilvægi þessara aðferða með því að fínþjálfa mállíkan á 1000 hágæða dæmum og sýnum fram á merkilega góða frammistöðu. Þjálfunargögnin samanstanda af vandlega völdum dæmum úr [uppruni gagna]. Við komumst að því að með örfáum sýnidæmum getur líkanið lært ákveðnar svartegundir og svarað flóknum fyrirspurnum eins og að gera ferðaáætlun og áætla hvernig mankynssagan væri ef einhver mikilvægur atburður hefði farið öðruvísi. Í samanburðarrannsókn með mannfólki sjáum við að þessi einfalda nálgun kemur líkaninu í sama gæðaflokk og betri spjallmenni sem eru í notkun. Þessar niðurstöður styðja tilgátu að nánast öll þekking í stórum mállíkönum er lærð í forþjálfuninni og það þarf ekki mikið af fínþjálfunargögnum til þess að kenna líkönum á einblína á ham sem samsvarar hágæða frálag.
```

Comment:
Góðar niðurstöður hafa fengist með því að þjálfa mállíkön án leiðsagnar, með mörgum fyrirmælum og styrkingarnámi.

Þessar niðurstöður styðja tilgátuna að... kemur úr forþjálfuninni... kenna líkönum að svara með hágæða svörum.